core/templates/services/state-top-answers-stats.service.ts

```diff
@@ -12,156 +12,147 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import { AnswerClassificationService } from
+  'pages/exploration-player-page/services/answer-classification.service';
+import { AnswerStats } from
+  'domain/exploration/AnswerStatsObjectFactory';
+import { InteractionRulesRegistryService } from
+  'services/interaction-rules-registry.service';
+import { StateTopAnswersStatsBackendApiService } from
+  'services/state-top-answers-stats-backend-api.service';
+import { States } from 'domain/exploration/StatesObjectFactory';
+
 /**
```

Comment:
Shouldn't the file overview be above the imports?

Reply:
Haha, vscode? Happens to me quite a lot (I use vscode).

Reply:
Nope I use vim, but I think it uses the same language-server that VSCode does IIRC? Either way it's the only negative thing I have to say about it, otherwise it's a dream to work with :)